artemis/config.py

```diff
@@ -698,6 +698,29 @@ class Nuclei:
                 "NUCLEI_TEMPLATE_CHUNK_SIZE is 200, three calls will be made with 200 templates each.",
             ] = get_config("NUCLEI_TEMPLATE_CHUNK_SIZE", default=200, cast=int)
 
+        class PlaceholderPagesHtmlElements:
+            PLACEHOLDER_PAGE_HTML_ELEMENTS: Annotated[
```

Comment:
IMO this is a good idea to extract this to a file (have config named "PLACEHOLDER_PAGE_CONTENT_FILENAME" and these contents being loaded from file)